src/parser/jobs/drg/modules/DragonSight.tsx

```diff
@@ -14,117 +19,158 @@ import DISPLAY_ORDER from './DISPLAY_ORDER'
 // give it a gcd for marking as truncated window
 const SHORT_WINDOW_BUFFER: number = 2500
 
+const EXPECTED_GCD_COUNT = 8
+
 // indicator for why the window was short (did you die or did your partner)
 const SHORT_WINDOW_FAULT = {
 	NONE: 0,			// everything's fine, no one's at fault
 	PARTNER: 1,
 	DRG: 2,
 }
 
-export default class DragonSight extends BuffWindowModule {
+// Override WindowEvaluator directly to allow the column to not show if all windows
+// have None status.
+class ShortWindowEvaluator implements WindowEvaluator {
+
+	private shortWindowReason: (window: HistoryEntry<EvaluatedAction[]>) => number
+
+	constructor(shortWindowReason: (window: HistoryEntry<EvaluatedAction[]>) => number) {
+		this.shortWindowReason = shortWindowReason
+	}
+
+	public suggest()  { return undefined }
+
+	public output(windows: Array<HistoryEntry<EvaluatedAction[]>>): EvaluationOutput | undefined {
+		const faults = windows.map(w => this.shortWindowReason(w))
+		if (faults.every(fault => fault === SHORT_WINDOW_FAULT.NONE)) {
+			return undefined
+		}
+		return {
+			format: 'notes',
+			header: {
+				header: <Trans id="drg.ds.notes.header">Short Window Cause</Trans>,
+				accessor: 'shortwindow',
+			},
+			rows: faults.map(fault => {
+				// check for a truncated window if not rushing, which would indicate that the tether partner (or the DRG) died
+				if (fault === SHORT_WINDOW_FAULT.PARTNER) {
+					return <Trans id="drg.ds.notes.partnerdied">Partner Died</Trans>
+				}
+
+				if (fault === SHORT_WINDOW_FAULT.DRG) {
+					return <Trans id="drg.ds.notes.drgdied">You Died</Trans>
+				}
+
+				return <></>
+			}),
+		}
+
+	}
+}
+
+export default class DragonSight extends BuffWindow {
 	static override handle = 'dragonsight'
 	static override title = t('drg.dragonsight.title')`Dragon Sight`
 	static override displayOrder = DISPLAY_ORDER.DRAGON_SIGHT
 
-	buffAction = ACTIONS.DRAGON_SIGHT
-	buffStatus = STATUSES.RIGHT_EYE
-	secondaryBuffStatus = STATUSES.RIGHT_EYE_SOLO
+	@dependency globalCooldown!: GlobalCooldown
 
-	deathTimes: number[] = []
+	buffAction = this.data.actions.DRAGON_SIGHT
+	override buffStatus = this.data.statuses.RIGHT_EYE
+	private secondaryBuffStatus = this.data.statuses.RIGHT_EYE_SOLO
 
-	override expectedGCDs = {
-		expectedPerWindow: 8,
-		suggestionContent: <Trans id="drg.ds.suggestions.missedgcd.content">
-			Try to land at least 8 GCDs during every <ActionLink {...ACTIONS.DRAGON_SIGHT} /> window.
-		</Trans>,
-		severityTiers: {
-			1: SEVERITY.MINOR,
-			2: SEVERITY.MEDIUM,
-			4: SEVERITY.MAJOR,
-		},
-	}
+	deathTimes: number[] = []
 
-	override trackedActions = {
-		icon: ACTIONS.DRAGON_SIGHT.icon,
-		actions: [
-			{
-				action: ACTIONS.CHAOS_THRUST,
-				expectedPerWindow: 1,
-			},
-			{
-				action: ACTIONS.FULL_THRUST,
-				expectedPerWindow: 1,
+	override initialise() {
+		super.initialise()
+
+		this.addEventHook({type: 'death', to: 'player'}, this.onDeath)
```

Comment:
breaking; this won't fire on the new event system

Reply:
Surprisingly, none of the DRG parses I tested pointed this out, and I was testing from new parses, not just top parses or something.